src/textual/message_pump.py

```diff
@@ -340,7 +340,7 @@ def set_interval(
         self._timers.add(timer)
         return timer
 
-    def call_after_refresh(self, callback: Callable, *args, **kwargs) -> None:
+    def call_after_refresh(self, callback: Callable, *args: Any, **kwargs: Any) -> None:
```

Comment:
`object` may be a better choice if you are going to type these.

Reply:
You think so?

Typing them as `object` means that you can't do _anything_ with them without having the type checkers complain.
I'll wait for you to comment further before determining whether I'll leave them as `Any` or whether to type them as `object`.